### PR TITLE
feat: add hovercards to list contributors page

### LINKED ISSUE #1953: Feature: Use Contributor hovercards on lists contributors page
### Type of feature

🍕 Feature

### Current behavior

On the contributors tab when viewing a list, we show avatars with no hovercard

image

### Suggested solution

Replace the static avatar with the `HovercardAvatar` from our design system

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Adds hovercards to the list contributors page.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Closes #1953

## Mobile & Desktop Screenshots/Recordings

**Before**

CleanShot 2023-10-20 at 17 09 57

**After**

CleanShot 2023-10-20 at 17 07 29

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: components/atoms/Avatar/avatar-hover-card.tsx [hunk 1]
```diff
@@ -10,27 +10,42 @@ import { getAvatarByUsername } from "lib/utils/github";
...
-      </HoverCard.Trigger>
-    </Link>
-    <HoverCard.Portal>
-      <HoverCard.Content sideOffset={5}>
-        <HoverCardWrapper username={contributor} repositories={repositories} />
-      </HoverCard.Content>
-    </HoverCard.Portal>
-  </HoverCard.Root>
-);
+const AvatarHoverCard = ({ contributor, repositories, size = "large" }: AvatarProps): JSX.Element => {
```
Comment (nickytonline) [inline components/atoms/Avatar/avatar-hover-card.tsx:16]: Added a size prop with the default being the original width and height of 500

File: components/molecules/ContributorListTableRow/contributor-list-table-row.tsx [hunk 2]
```diff
@@ -155,14 +156,7 @@ const ContributorListTableRow = ({
...
         {/* Column: Contributors */}
-        <div className={clsx("flex-1 lg:min-w-[12.5rem] overflow-hidden")}>
-          <DevProfile
-            company={user?.company || getLastContributedRepo(data)}
-            username={login}
-            hasBorder={!contributor.author_login}
-          />
-        </div>
-
+        <AvatarHoverCard contributor={contributor.username} repositories={[]} size="small" />
```
Comment (nickytonline) [inline components/molecules/ContributorListTableRow/contributor-list-table-row.tsx:159]: I'm currently not passing any repos and it still works and lists 3 repos (max) as per the hover card constraints.

I can see about adding repos in the array here, but wonder if it's necessary.

**Review by jpmcb — APPROVED** [review]
Looks slick! Nice work!!

File: components/molecules/ContributorListTableRow/contributor-list-table-row.tsx (hunk 2 above)
Comment (OgDev-01) [reply components/molecules/ContributorListTableRow/contributor-list-table-row.tsx:159]: not necessary in this context i'd say... we can ship as is

**Review by OgDev-01 — APPROVED** [review]
LGTM

**Pull request merged**